packages/framework/client-logger/app-insights-logger/jest.config.js

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
```

Comment:
Do we need jest in this package? Can we get away with just using `mocha` w/ unit tests? We should also add some simple unit tests if we can (packages with no tests always make me nervous).

Reply:
Switched from mocha to jest and added unit tests